extension-copy-svgs.js

```diff
@@ -17,11 +17,7 @@ const copyDoc = async (projectPath) => {
  */
 const copyFiles = async (argumentsArray) => {
   let projectPath = '';
```

Comment:
NIT: I think you can declare the projectPath variable as `const projectPath = 'dist/apps/vs-code-designer';` since the beginning as it will not change anymore. And remove the argumentsArray

Reply:
Done!